plugins/TranslationServer.js

```diff
@@ -21,6 +21,8 @@ var availableTranslations = [
 var HOOT_HOME = process.env.HOOT_HOME;
 if (typeof hoot === 'undefined') {
     hoot = require(HOOT_HOME + '/lib/HootJs');
+    hoot.Log.setLogLevel("warn");
+    hoot.Settings.set({"osm.map.reader.xml.add.child.refs.when.missing":"true"});
```

Comment:
Make sure that the translation server keeps missing nodes so that geometry doesn't need to be fully hydrated in order to translate correctly.

Reply:
We also set some hoot.Settings here https://github.com/ngageoint/hootenanny/blob/d7bde420b931a8260f7464d3514a2fbab80a5786/plugins/TranslationServer.js#L264-L266

Think we should move them up to this spot?